src/components/file/DropzoneHelper.tsx

```diff
@@ -0,0 +1,77 @@
+import { useContext, useRef } from 'react';
+
+import FolderOutlinedIcon from '@mui/icons-material/FolderOutlined';
+import UploadFileOutlinedIcon from '@mui/icons-material/UploadFileOutlined';
+import { Button, Stack, Typography } from '@mui/material';
+
+import { useBuilderTranslation } from '@/config/i18n';
+import { BUILDER } from '@/langs/constants';
+
+import { UppyContext } from './UppyContext';
+
+const DropzoneHelper = (): JSX.Element => {
+  const { t } = useBuilderTranslation();
+  const { uppy } = useContext(UppyContext);
+  const ref = useRef<HTMLInputElement>(null);
+
+  const handleClick = () => {
+    const { current } = ref;
+    if (current) {
+      current.click();
+    }
+  };
+
+  const handleFiles = () => {
+    const { current } = ref;
+    if (current) {
+      const { files } = current;
+      if (files) {
+        // add files selected to uppy, this will upload them
+        [...files].map((file) =>
+          uppy?.addFile({ data: file, name: file.name }),
+        );
+      } else {
+        console.error('no files found !');
+      }
+    }
+  };
+
+  return (
+    <Stack
+      justifyContent="flex-start"
+      alignItems="center"
+      direction="column"
+      flexGrow={1}
+      spacing={3}
+      sx={{ height: '100%', marginTop: '10%' }}
+    >
+      <UploadFileOutlinedIcon sx={{ fontSize: '50px', color: '#5050d2' }} />
```

Comment:
MUI Icons support specifying the use of colours from the palette:
```suggestion
      <UploadFileOutlinedIcon color="primary" sx={{ fontSize: '50px' }} />
```

Also we have the primary graasp colour string stored in a constant in case you need. But using the theme is the preffered approach.